src/cfe.h

```diff
@@ -1,24 +1,52 @@
-/** @file */
 /** @file
+ *  @brief Declare variables for internal storage of master Code/Flag
+ *  table entries.
+ */
+
+/**
+ *  This header file contains the structure and variable declarations
+ *  used to store master Code/Flag table entries internally.
+ *
+ *  @author J. Ator
+ *  @date 2017-11-16
  */
 
 #define MAX_MEANING_LEN 150
 
 struct code_flag_entry {
     f77int iffxyn;
-	/* Bitwise representation of FXY number to which this entry belongs. */
+    /** @var iffxyn
```

Comment:
I think you can do this same thing, without the var tag, using a special start for the comment. Like this:

`f77int iffxyn; /*< Bitwise representation of FXY number to which this entry belongs. */`

Reply:
You may be right, but I actually prefer the @var tag because it's more descriptive than using a special "<" character.